requirements.txt

```diff
@@ -4,8 +4,8 @@ Pillow
 cython
 matplotlib
 scikit-image
-tensorflow>=1.3.0
-keras>=2.0.8
+tensorflow==1.13.1
+keras==2.0.8
```

Comment:
I think keras is safe to remove, since all instances are part of the tensorflow module

Reply:
It's my first try, so I make a lot of mistakes. Thank you so much for your review :)